### ISSUE TITLE
Unexpected error results

### ISSUE THREAD

**Issue by erosman** [UziTech/eslint-linter-browserify#517]
re: https://cdn.jsdelivr.net/npm/eslint-linter-browserify/linter.min.js

Is that the latest ESLint?
`eslint.Linter.version` shows "8.41.0" 

I am getting unexpected errors while testing. What can be the reason?

```js
const linter = new eslint.Linter();
const messages = linter.verify("const a = 5;", {
  rules: {
    semi: ["error", "never"]
  }
}, { filename: "foo.js" });

console.log(messages);
```
Result
```json
[
  {
    "ruleId": null,
    "fatal": true,
    "severity": 2,
    "message": "Parsing error: The keyword 'const' is reserved",
    "line": 1,
    "column": 1,
    "nodeType": null
  }
]
```

**Comment by erosman** [comment]
Sorry... I was testing with an older version (downloaded before) for some reason.

**Comment by erosman** [comment]
For reference, the error can be fixed with `env: {es6: true}`.


```js
const linter = new eslint.Linter();
const messages = linter.verify("const a = 5;", {
  env: {es6: true},
  rules: {
    semi: ["error", "never"]
  }
}, { filename: "foo.js" });

console.log(messages);
```
